variables.tf

```diff
@@ -1,140 +1,140 @@
+    policy = string
+  }))
+  default = []
 }
-variable "security_group_ids" {
-  type        = list(string)
-  description = "Only required if a schedule_expression is set; default is nothing.  Will create an outbound permissive SG if none is provided."
-  default     = []
+
+variable "task_container_definitions" {
+  description = "Required: A json encoded string containing a set of container definitions for the given Fargate task."
+  type        = string
+  validation {
+    condition     = length(var.task_container_definitions) > 0
+    error_message = "task_container_definitions -> must container at least 1 container definition"
+  }
 }
-variable "assign_public_ip" {
-  type        = bool
-  description = "Set to true if subnet is 'public' with IGW, false is subnet is 'private' with NAT GW. Defaults to true, as required by default vpc"
-  default     = true
+
+variable "task_ephemeral_storage_size_in_gib" {
+  description = "Optional: The ephemeral_storage size in GiB that is allocated to the task at runtime."
+  type        = number
+  default     = 0
 }
-variable "log_retention_in_days" {
-  type        = string
-  description = "Optional; The number of days you want to retain log events in the log group.  Defaults to 60"
-  default     = "60"
+
+variable "task_count" {
+  description = "Optional: The number of task to start when the event is triggered."
+  type        = number
+  default     = 1
 }
-variable "log_group_name" {
+
+variable "task_container_overrides" {
+  description = "A json encoded string reflecting a set of command layer overrides. `containerOverrides = [{name='container-name',command=['command']}]`"
   type        = string
-  description = "Optional; The name of the log group. By default the `name` variable will be used."
   default     = ""
 }
-variable "log_group_stream_prefix" {
-  type        = string
-  description = "Optional; The name of the log group stream prefix. By default this will be `container`."
-  default     = "container"
-}
-variable "log_group_region" {
+
+#
+# ECS
+#
+
+variable "ecs_cluster_arn" {
+  description = "Required: The designated ECS cluster ARN in which the task will run."
   type        = string
-  description = "Optional; The region where the log group exists. By default the current region will be used."
-  default     = ""
 }
-variable "efs_configs" {
-  type = list(object({
-    container_name = string
-    file_system_id = string
-    root_directory = string
-    container_path = string
-  }))
-  description = "Optional; List of {file_system_id, root_directory, container_path} EFS mounts."
-  default     = []
+
+variable "ecs_execute_command_enabled" {
+  description = "Optional: Add statements to the IAM role of the task allowing `aws ecs execute-command' commands to function properly. By default is false."
+  type        = bool
+  default     = false
 }
+
 variable "ecs_platform_version" {
+  description = "Optional: The ECS platform version used to lauch the Fargate container."
```

Comment:
description = "Optional: The ECS platform version used to lauch the Fargate container."

launch*